src/lib/websocket.erl

```diff
@@ -131,7 +131,7 @@ decode(Data) ->
 %%% Internal functions
 %%%===================================================================
 gen_accept_key() ->
-    random:seed(erlang:now()),
+    rand:seed(erlang:now()),
```

Comment:
What's the point of trying to seed here? crypto:strong_rand_bytes/1 takes care of that, no?